File: Python/Practice Python/10/2.py
sum = int(input("Enter Value : "))
add = False
n = [5,7,1,2,8,4,3]

for num in n:
    for nums in n:
        if (num + nums) == sum and num != nums and num > nums:
            print(num,'+',nums,'=',sum)
            add = True
if not add : 
    print("No Number Adds to",sum)

File: C/Chapters/chapter 4/problem set/2.c
# include <stdio.h>

int main()
{   int n;
    int sum = 0;
    printf("Enter a Number: ");
    scanf("%d",&n);
    for(int i = 1; i <= n; i++){
        sum += i;
    }

    printf("The Sum of First %d Natural no. is %d",n,sum);
    
    return 0;
}